src/nwp_consumer/internal/inputs/ecmwf/_models.py

```diff
@@ -0,0 +1,19 @@
+import datetime as dt
+from dataclasses import dataclass
+
+import nwp_consumer.internal as internal
+
+
+@dataclass
+class ECMWFMarsFileInfo(internal.FileInfoModel):
+    inittime: dt.datetime
+    area: str
+
+    def filename(self) -> str:
+        return f"ecmwf_{self.area}_{self.inittime.strftime('%Y%m%dT%H%M')}.grib"
```

Comment:
could you give an example of one

Reply:
Done in https://github.com/openclimatefix/nwp-consumer/commit/181bf9789f7cccfc720d3a56d0a807a5d6a4cf26